### DISCUSSION TITLE
Execution Engine version pinning in pipelines?

### DISCUSSION THREAD

**Discussion by korteksz** [nuke-build/nuke#1461] [Q&A]
A new execution engine has been published (9.0.1) which seems to have some incompatible changes with my currently used package version s (8.1.4):

> MissingMethodException: Method not found: 'System.Collections.Generic.IReadOnlyCollection`1<Nuke.Common.Tooling.Output> Nuke.Common.Tools.DotNet.DotNetTasks.DotNet(Nuke.Common.Tooling.ArgumentStringHandler, System.String, System.Collections.Generic.IReadOnlyDictionary`2<System.String,System.String>, System.Nullable`1<Int32>, System.Nullable`1<Boolean>, System.Nullable`1<Boolean>, System.Action`2<Nuke.Common.Tooling.OutputType,System.String>, System.Action`1<Nuke.Common.Tooling.IProcess>)'.

I would like to temporarily pin the version of this execution tool back to version 8.x. Locally it is clear that I install the specific version of the Nuke dotnet tool, but I don't find a way of defining this version in my GitHub pipelines. Can someone help me out how this is possible? 

Thanks in advance,

Zoltán

**Comment by matkoch** [comment]
Without showing your invocation it's impossible to give any advice. And I'm not sure about pinning either. `Nuke.GlobalTool` isn't meant for pipeline invocations.
